feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -64,9 +66,13 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
     switchMap((cartId) => this.multiCartFacade.getCartEntity(cartId))
   );
 
+  // Flag to prevent cart loading when logged in with code flow
+  protected shouldLoad = true;
```

Comment:
Also `shouldLoad` seems to be too general name in my opinion. 
Can you propose something more descriptive? 

`shouldLoadCartOnCodeFlow`?

Reply:
Changed